classpath/javax-net-ssl.cpp

```diff
@@ -0,0 +1,7 @@
+// javax-net-ssl.cpp
+#include "jni.h"
+#include "sslcontext.h"
+
+extern "C" JNIEXPORT SSLCONTEXT JNICALL Java_javax_net_ssl_SSLContext_create(JNIEnv* env, jclass) {
```

Comment:
Pretty sure this needs to return jlong, and then you need to return the pointer to the ctx, not the actual ctx.